matter_server/server/device_controller.py

```diff
@@ -155,18 +157,24 @@ async def commission_with_code(self, code: str) -> MatterNodeData:
 
         # perform a quick delta sync of certificates to make sure
         # we have the latest paa root certs
+        LOGGER.info("Commissioning new device with code.")
```

Comment:
this is duplicate

Reply:
It is before fetching the certificate... What if certificate fetching is slow, and causes the commissioning to break? I think it is really beneficial to see what is happening during commissioning to pinpoint problems.

If we drop the certificate fetching, then I am ok with removing this.

Reply:
Yeah, my local branch with a bunch of patches is almost ready which drops this certificate fetching (as it doesnt work anyways, only at startup)